community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md

```diff
@@ -0,0 +1,99 @@
+
+## Azure Key Vault
+
+Another module we created, was specifically designed to collect functionality related to the Azure Key Vault.
+We figured there's a lot of extra functionality that can be useful during interacting with this Azure resource.
+
+The `Arcus.Scripting.KeyVault` PowerShell module contains now only two functions: `Get-AzKeyVaultAccessPolicies` and `Set-AzKeyVaultSecretFromFile` but is for sure going to expand in more functionality.
+
+Following example shows how we can get all the access policies on the Azure Key Vault into a PowerShell object.
+
+```powershell
+PS> $accessPolicies = Get-AzKeyVaultAccessPolicies -KeyVaultName "arcus-key-vault"
+# accessPolicies: {list: [{tenantId: ...,permissions: ...}]}
+```
+
+See the dedicated [docs](https://scripting.arcus-azure.net/features/powershell/azure-key-vault) pages for more info on this subject.
+
+## Azure Data Factory
+
+One of the approaches we would like to make for better user-friendliness, is splitting enable/disable, setting/getting, start/stop... functionality each time in 2 different functions instead of a single function that does both accourding to given arguments. This will make things much clearer and simpler.
+
+A good example of this, is how we created the `Arcus.Scripting.DataFactory` PowerShell module. This module has two functions to enable and disable a trigger of an Azure Data Factory pipeline: `Enable-AzDataFactoryTrigger` and `Disable-AzDataFactoryTrigger` instead of the original `EnableDisableDataFactoryTrigger`.
+
+Following example shows how we can enable the Azure Data Factory trigger:
+
+```powershell
+PS> Enable-AzDataFactoryTrigger -ResourceGroupName "arcus-resource-group" -DataFactoryName "arcus-data-factory-name" -DataFactoryTriggerName "arcus-data-factory-trigger-name"
+# The trigger 'my-data-factory-trigger-name' has been enabled.
+```
+
+See the [docs](https://scripting.arcus-azure.net/features/powershell/azure-data-factory) for more information on these two functions.
+
+## Azure (Table) Storage
+
+The `Arcus.Scripting.Storage.Table` PowerShell module is the start of a whole bunch of storage-related scripts that will in the future be collected in a `Arcus.Scripting.Storage` PowerShell module. For now, we only have the Azure Table Storage module which already contains the funcctionality to create a new table inside an Azure Storage Account.
+The `Create-AzStorageTable` also has the ability to re-create an existing table if that is required.
+
+```powershell
+PS> Create-AzStorageTable -ResourceGroupName "stock" -StorageAccountName "admin" -TableName "products"
+# Creating table 'products' in the storage account 'admin'..."
+```
+
+See the [docs](https://scripting.arcus-azure.net/features/powershell/azure-storage) for more information on this function.
+
+## Azure Resource Manager (ARM)
+
+The `Arcus.Scripting.ARM` PowerShell module contains the capability to inject content into a ARM template, controling the formatting of the injected content.
```

Comment:
Perhaps include a bit of info as to what users will be able to use the module for in general?

```suggestion
The `Arcus.Scripting.ARM` PowerShell module has been built to help you in manipulating ARM templates before triggering a deployment, allowing you to reduce the complexity of these templates.  
Currently, this module contains the capability to inject content into an ARM template, controling the formatting of the injected content.
```